packages/mobile/src/account/Contact.tsx

```diff
@@ -0,0 +1,170 @@
+import Button, { BtnTypes } from '@celo/react-components/components/Button'
+import KeyboardSpacer from '@celo/react-components/components/KeyboardSpacer'
+import Switch from '@celo/react-components/components/Switch'
+import TextInput from '@celo/react-components/components/TextInput'
+import colors from '@celo/react-components/styles/colors'
+import fontStyles from '@celo/react-components/styles/fonts'
+import { anonymizedPhone } from '@celo/utils/src/phoneNumbers'
+import React, { useCallback, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { ScrollView, StyleSheet, Text, View } from 'react-native'
+import DeviceInfo from 'react-native-device-info'
+import Mailer from 'react-native-mail'
+import SafeAreaView from 'react-native-safe-area-view'
+import { useSelector } from 'react-redux'
+import { getE164PhoneNumber } from 'src/account/selectors'
+import { CELO_SUPPORT_EMAIL_ADDRESS } from 'src/config'
+import i18n, { Namespaces } from 'src/i18n'
+import { headerWithBackButton } from 'src/navigator/Headers'
+import Logger from 'src/utils/Logger'
+
+interface Email {
+  subject: string
+  recipients: [string]
+  body: string
+  isHTML: boolean
+  attachment?: {
+    path: string
+    type: string
+    name: string
+  }
+}
+
+const Support = () => {
+  const { t } = useTranslation(Namespaces.accountScreen10)
+  const [message, setMessage] = useState('')
+  const [attachLogs, setAttachLogs] = useState(true)
+  const e164PhoneNumber = useSelector(getE164PhoneNumber)
+
+  const sendEmail = useCallback(async () => {
+    const deviceInfo = {
+      version: DeviceInfo.getVersion(),
+      buildNumber: DeviceInfo.getBuildNumber(),
+      apiLevel: DeviceInfo.getApiLevelSync(),
+      deviceId: DeviceInfo.getDeviceId(),
+    }
+    const userId = anonymizedPhone(e164PhoneNumber)
+    const emailSubject = 'Celo support for ' + (userId || 'unknownUser')
+    const email: Email = {
+      subject: emailSubject,
+      recipients: [CELO_SUPPORT_EMAIL_ADDRESS],
+      body: `${message}<br/><br/><b>${JSON.stringify(deviceInfo)}</b>`,
+      isHTML: true,
+    }
+    if (attachLogs) {
+      const combinedLogsPath = await Logger.createCombinedLogs()
```

Comment:
This can take some time on slow phones. Can we show a spinner or something while this is working?